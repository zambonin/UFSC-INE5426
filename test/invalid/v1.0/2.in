int i
i = ref 0
