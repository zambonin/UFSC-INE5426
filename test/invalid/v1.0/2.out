[Line 2] semantic error: reference operation expects a pointer
